src/PostRector/Rector/NameImportingPostRector.php

```diff
@@ -29,6 +34,12 @@ public function __construct(
     ) {
     }
 
+    public function beforeTraverse(array $nodes)
+    {
+        $this->currentUses = $this->useImportsResolver->resolve();
```

Comment:
We calculate the current use statements so that they don't need to be calculated for every node